deploy_pulp3/deploy-pulp3.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: localhost
```

Comment:
Can this *not* be localhost? What if I want to install Pulp on, say, some other machine? Let this be `hosts: all`, and have the user specify the hosts when calling this playbook.

Reply:
yes this is how it is going to be once the 'wip' label will be removed ;)

Reply:
Great. :+1: